CMakeLists.txt

```diff
@@ -2219,29 +2217,49 @@ if(FAAD)
   endif()
 endif()
 
+# FDK-AAC is loaded dynamically at runtime by EncoderFdkAac using QLibrary,
+# so copy it into the Windows and macOS packages, but do not link to it.
+find_library(FDK_AAC_LIBRARY fdk-aac)
+# On Windows find_library finds the .lib file, but the installer needs the .dll file.
```

Comment:
Why we search for the lib in the fist place?

Reply:
The comments explain that clearly...